templates/core/login.mustache

```diff
@@ -237,9 +237,16 @@
         var cookieRibbon = document.getElementById('jsCookieRibbon');
         var cookieBtn = document.getElementById('jsCookieBtn');
 
-        $('#jsCookieBtn').click(function() {
+        // Don't show ribbon if cookie permission granted
+        if(localStorage.getItem("cookie-enable")=="1"){
```

Comment:
Do we need to prefix the localStorageItem? cookie-enabled seems a bit generic but I'm not familiar with local storage so not sure if it is dmain specific and therefore already prefixed in that way?

Reply:
Valid question raised @cehwitham ! 

I discussed a little with @Android63 as well as looked up online and it seems that localStorage keys are after all domain-specific.

Check this link - https://stackoverflow.com/questions/4201239/in-html5-is-the-localstorage-object-isolated-per-page-domain
And this  - https://en.wikipedia.org/wiki/Same-origin_policy#Origin_determination_rules

So it works on Protocol-Host-Port theory and if we have more Nightingale based themes using same key/flag "cookie-enable" in localStorage, as long as they belong to different domain, we are good!

If you still think it's wise for me to rename it to "nightingale-vc-cookie-enable" or something similar, I could do so but I don't see any reason for having conflict with using same generic name across all sites.